Makefile

```diff
@@ -71,6 +72,9 @@ gen-api: $(GOBINPATH)/swagger ## Run the go-swagger code generator
 	$(GOBINPATH)/swagger generate client -q -A lakefs -f ./swagger.yml -P models.User -t $(API_BUILD_DIR)
 	$(GOBINPATH)/swagger generate server -q -A lakefs -f ./swagger.yml -P models.User -t $(API_BUILD_DIR) --exclude-main
```

Comment:
I think this will now happen automagically when you `go generate` because of the top of `api/handler.go`.  Best not leave _both_, they will just drift apart and debugging that will hurt.